samples/javascript_nodejs/57.teams-conversation-bot/bots/teamsConversationBot.js

```diff
@@ -180,39 +174,42 @@ class TeamsConversationBot extends TeamsActivityHandler {
     async messageAllMembersAsync(context) {
         const members = await this.getPagedMembers(context);
 
-        members.forEach(async (teamMember) => {
+        await Promise.all(members.map(async (member) => {
```

Comment:
Same issue with regard to `forEach` with async functions. Use `Promis.all` to synchronize.